squad/frontend/views.py

```diff
@@ -476,7 +476,28 @@ def test_run_suite_test_details(request, group_slug, project_slug, build_version
     test_name = test_name.replace("$", "/")
     suite_slug = suite_slug.replace("$", "/")
     metadata = get_object_or_404(SuiteMetadata, kind='test', suite=suite_slug, name=test_name)
-    test = get_object_or_404(context['test_run'].tests, suite=context['suite'], metadata=metadata)
+    tests = Test.objects.filter(suite=context['suite'], metadata=metadata)
```

Comment:
I think it's missing `context['test_run'].tests` no?

Reply:
Not 100% sure. Does it go into the same test_run if it comes from a resubmitted test job?

Reply:
no, each test job creates a new test run

Reply:
then it's ok, this should not include it in the filtering

Reply:
Hum, then you'd need to filter by `testrun.build` [and maybe by `testrun.environment`?], otherwise you'll be getting tests from the same suite for all builds/envs.